packages/client-sdk-nodejs/src/internal/leaderboard-data-client.ts

```diff
@@ -522,4 +535,11 @@ export class LeaderboardDataClient implements ILeaderboardDataClient {
       );
     });
   }
+
+  protected getNextDataClient(): leaderboard.LeaderboardClient {
```

Comment:
this method is stolen from the existing code for the CacheClient where we are already supporting multiple grpc connections.